lib/cli/commands/unlock.ts

```diff
@@ -11,11 +11,15 @@ export const builder = {};
 
 const formatOutput = (response: UnlockNodeResponse.AsObject) => {
   console.log('xud was unlocked successfully');
+  const readyClients = response.unlockedLndsList;
+  if (response.connextReady) {
+    readyClients.push('ETH');
+  }
   if (response.unlockedLndsList.length) {
-    console.log(`The following wallets were unlocked: ${response.unlockedLndsList.join(', ')}`);
+    console.log(`The following wallets are unlocked and ready: ${readyClients.join(', ')}`);
   }
   if (response.lockedLndsList.length) {
-    console.log(`The following wallets could not be unlocked: ${response.lockedLndsList.join(', ')}`);
+    console.log(`The following wallets could not be unlocked: ${readyClients.join(', ')}`);
```

Comment:
it is incorrect handling of `could not be unlocked` case.
1. This case means that something wrong with L2 client and it can not be unlocked (`response.lockedLndsList` was used because of that before).
2. But for now it looks like `could not be unlocked` = `client already unlocked` as a result it leads to incorrect erroe handling (my lnd can not be unlocked because something wrong with my docker network):
![Screenshot from 2020-10-29 21-30-59](https://user-images.githubusercontent.com/29906866/97617638-11d6b180-1a2f-11eb-8372-5d78e3a9698f.png)

Reply:
Thanks for catching this, it was an oversight on my part. If a client was already unlocked before the call, it should show up under the list of `The following wallets are unlocked and ready` - along with the clients that were unlocked when we made this call. Only if the client could not be unlocked (due to being offline, wrong password, etc...) should it show up as `could not be unlocked`.